doc/source/PreConfiguredSites.rst

```diff
@@ -402,7 +395,7 @@ The following is required for building new spack environments and for using spac
    module load ecflow/5.8.4
 
 .. note::
-   Support for spack-stack-1.4.0 will be added later on develop. The instructions below are for an older release.
+   Support for spack-stack-1.4.1 will be added later on develop. The instructions below are for an older release.
```

Comment:
Lines 398-417 could be removed and replaced with:

```
For ``spack-stack-1.4.1`` with Intel, load the following modules after loading miniconda and ecflow:

   module use /contrib/EPIC/spack-stack/spack-stack-1.4.1/envs/unified-dev/install/modulefiles/Core
   module load stack-intel/2021.3.0
   module load stack-intel-oneapi-mpi/2021.3.0
   module load stack-python/3.9.12
   module available
```

1.4.1 is installed on AWS, Azure, and GCP (by @mark-a-potts)

Reply:
Please re-review https://github.com/JCSDA/spack-stack/pull/669/commits/51ba4b8ed12223d73a81217f89dcb7b9038ddf4b